cgi-bin/LJ/CleanHTML.pm

```diff
@@ -1900,6 +1886,10 @@ sub convert_user_mentions {
         !mge;
 }
 
+# Keys available in opts hashref:
```

Comment:
Kind of makes you want for types or something.

Reply:
😑 srsly